syntax = "proto2";

package proto.db;

message query_command {
	required string table_name = 1;
	required string where_clause = 2;
	repeated bytes args = 3;
}